tests/xrefs.py

```diff
@@ -1,6 +1,6 @@
 """
 Blah.
-`undefined_xref_from_module`
+L{undefined_xref_from_module}
```

Comment:
This seems to have undone a lot of the previous formatting changes

Reply:
Yes, this file is there to specifically test the docxref parsing code in pyflyby, so the changes should not have been made there.

Reply:
Ahh makes sense